apps/src/code-studio/components/pairing/StudentSelector.jsx

```diff
@@ -65,6 +65,7 @@ export default class StudentSelector extends React.Component {
         <div className="clear" />
         {this.state.selectedStudentIds.length !== 0 && (
           <button
+            style={{marginLeft: 0}}
```

Comment:
nit: our [styleguide](https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md) prefers using a styles object instead of inline styles.